code/api/tests/test_posts.py

```diff
@@ -18,7 +18,7 @@ def create_post(title, author):
             title=title,
             description="testDesc",
             content_type=LocalPost.ContentType.PLAIN,
-            content="testContexxt",
+            content=b64.b64encode("testContexxt".encode("utf-8")),
```

Comment:
I fixed the failed test by removing the b64encoding. Why do we need to encode here? Won't data sent already be encoded if it's supposed to be encoded (like an image) and other data doesn't need to be encoded at all?

Reply:
![image](https://user-images.githubusercontent.com/47841864/142777910-13035ca8-5a65-4e57-8c87-55032d68c2ae.png)
 
Image for reference. Data is getting encoded when it shouldn't be. So the post received by another author via their inbox is garbeled. Remove those b64encodes and I think this should be fixed.